mingw-w64-rustup/PKGBUILD

```
@@ -0,0 +1,101 @@
+_realname=rustup
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.28.1
+pkgrel=1
+pkgdesc="The Rust toolchain installer (mingw-w64)"
+arch=('x86_64')
+mingw_arch=('mingw64' 'ucrt64' 'clang64')
+url="https://github.com/rust-lang/rustup"
+license=('spdx:Apache-2.0 AND MIT')
+msys2_references=(
+	'archlinux: rustup'
+)
+depends=(
+	"${MINGW_PACKAGE_PREFIX}-curl"
+	"${MINGW_PACKAGE_PREFIX}-xz"
+	"${MINGW_PACKAGE_PREFIX}-zstd"
+)
+makedepends=(
+	"git"
+	"${MINGW_PACKAGE_PREFIX}-rust"
+	"${MINGW_PACKAGE_PREFIX}-rust-bindgen"
+	"${MINGW_PACKAGE_PREFIX}-cmake"
+	"${MINGW_PACKAGE_PREFIX}-nasm"
+)
+optdepends=(
+	"${MINGW_PACKAGE_PREFIX}-gdb: rust-gdb script"
+	"${MINGW_PACKAGE_PREFIX}-lldb: rust-lldb script"
+)
+provides=("${MINGW_PACKAGE_PREFIX}-rust")
+conflicts=("${MINGW_PACKAGE_PREFIX}-rust")
+install='post.install'
+options=("!lto")
+source=(
+	"rustup-${pkgver}.tar.gz::${url}/archive/${pkgver}.tar.gz"
+	"git+https://github.com/aws/aws-lc-rs#tag=aws-lc-sys/v0.27.1"
+	"git+https://github.com/aws/aws-lc.git"
+	"aws-lc-clang-build.patch" # stolen from https://github.com/msys2/MINGW-packages/blob/911c06155de002672b27b3c8bbe893615250b6bf/mingw-w64-zed/aws-lc-clang-build.patch
```

```suggestion
	"aws-lc-clang-build.patch" # taken from https://github.com/msys2/MINGW-packages/blob/911c06155de002672b27b3c8bbe893615250b6bf/mingw-w64-zed/aws-lc-clang-build.patch
```

better use neutral langauge :D